### DISCUSSION TITLE
Did you define what 'tools' are yourself?

### DISCUSSION THREAD

**Discussion by PaterFrog** [TrueHorse/Wild-Tool-Access#24] [General]
I was wondering if it was possible to exclude swords from the tool hotbar - they're already in the swords hotbar, after all.

**Comment by TrueHorse** [comment]
That would probably make sense. In general the mod would probably benefit from a semi-big overhaul. For example that every category is defined by a file, like stuff is.
Currently everything except stuff is defined by what items inherit from a certain Minecraft class, in this case from ToolItem.
But despite that, expect me to not touch this mod in the forseeable future. Currently not realy motivated to work on it and also have the one or other other thing to do, sorry.
